src/lua/CommunityBalanceMod/FortressPvE/TeleportMixin.lua

```diff
@@ -0,0 +1,21 @@
+
+
+function TeleportMixin:GetCanTeleport()
+
+    local canTeleport = true
+    if self.GetCanTeleportOverride then
+        canTeleport = self:GetCanTeleportOverride()
+    end
+
+    --new
+    if self:GetTechId() == kTechId.FortressShift 
```

Comment:
Instead of overriding this function, could we add the GetCanTeleportOverride function to the Fortress structure classes? Think it would be a bit cleaner rather than having to maintain this list

Reply:
works great with overriding now